api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -319,7 +320,7 @@ public long getQueryBatchSize() {
                     offsetDateTimeUtc(rs.getTimestamp("data_use_agreement_bypass_time")))
                 .dataUseAgreementCompletionTime(
                     offsetDateTimeUtc(rs.getTimestamp("data_use_agreement_completion_time")))
-                .dataUseAgreementSignedVersion(rs.getInt("data_use_agreement_signed_version"))
+                .dataUseAgreementSignedVersion(rs.getInt("ducc_signed_version"))
```

Comment:
changing this name would be a larger effort.  That's a TODO for future Reporting work.